zerver/lib/export.py

```diff
@@ -1266,11 +1266,11 @@ def convert_to_id_fields(data: TableData, table: TableName, field_name: Field) -
         item[field_name + "_id"] = item[field_name]
         del item[field_name]
 
-def re_map_foreign_keys(data: TableData,
-                        table: TableName,
+def re_map_foreign_keys(data_table: List[Record],
                         field_name: Field,
                         related_table: TableName,
-                        verbose: bool=False) -> None:
+                        verbose: bool=False,
+                        id_field: bool=False) -> None:
```

Comment:
The 'id_field' bool variable is added to the function just to check if the field is the ID of that object, and not the foreign key relation.
For foreign key field names, a "_id" has to be added after the field name, however we don't need that for the ID field of the object.